Added installation instructions

georgebrindeiro opened the pull request.

georgebrindeiro committed:
Added installation instructions

emersonlaurentino commented:
This mode of installation is while the application is being analyzed in the Chrome Web Store. 

It would be nice to let that explain, because in the future it's just a button. 

@georgebrindeiro can you do that?

georgebrindeiro committed:
Added disclaimer about installation

This is only a temporary process while Chrome Web Store reviews the app for publishing

georgebrindeiro-vtex commented:
Done!

emersonlaurentino commented:
Thanks!
